user_docs/en/userGuide.t2t

```diff
@@ -292,6 +293,7 @@ For example, if you are typing into an editable text field, the editable text fi
 
 The most common way of navigating around Windows with NVDA is to simply move the system focus using standard Windows keyboard commands, such as pressing tab and shift+tab to move forward and back between controls, pressing alt to get to the menu bar and then using the arrows to navigate menus, and using alt+tab to move between running applications.
 As you do this, NVDA will report information about the object with focus, such as its name, type, value, state, description, keyboard shortcut and positional information.
+When the [NVDA Highlighter #VisionNVDAHighlighter] is enabled, the location of the current system focus is also exposed visually.

```

Comment:
See one of my previous comments about the name, which is now Focus Highlight in the gui.